app/src/frontend/app.tsx

```diff
@@ -112,6 +113,7 @@ class App extends React.Component<AppProps, AppState> {
                 <Route exact path="/data-accuracy.html" component={DataAccuracyPage} />
                 <Route exact path="/data-extracts.html" component={DataExtracts} />
                 <Route exact path="/contact.html" component={ContactPage} />
+                <Route exact path="/history.html" component={ChangesPage} />
```

Comment:
The history page is accessible but there is currently no link pointing to it, given its very rough form.